pkg/sql/sem/tree/pgwire_encode.go

```diff
@@ -162,6 +170,27 @@ func init() {
 	}
 }
 
+// PgwireFormatFloat returns a []byte representing a float according to
+// pgwire encoding. The result is appended to the given buffer.
+func PgwireFormatFloat(
```

Comment:
wonder if this should be PGWire, but that's an optional nit

Reply:
i was matching the naming of PgwireFormatBool